src/app/review-rating/review-rating.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
```

Comment:
We don't need a ReviewRatingModule, since we just use the review rating component in notification service